softlayer/vm/softlayer_vm.go

```diff
@@ -426,3 +426,18 @@ func (vm SoftLayerVM) postCheckActiveTransactionsForDeleteVM(softLayerClient sl.
 
 	return nil
 }
+
+func (vm SoftLayerVM) ExecCommand(virtualGuest datatypes.SoftLayer_Virtual_Guest, command string) (string, error) {
```

Comment:
Hey, do these methods need to be public? Could we make them private? Usually public methods should be part of the interface... I would rather have these private if possible, especially, since we will likely refactor this functionality/code into the bosh-agent. Thanks.